content/en/docs/getting-started/kubernetes-quickstart.md

```diff
@@ -0,0 +1,209 @@
+* K3s
+* Kubectl
+
+### Windows and Linux requirements
+* Virtualbox
+* Vagrant
+* Helm
+* K3s
+* Kubectl
+
+## Create a Kubernetes cluster
+
+### Create a K3s Cluster on Mac with Lima
+
+* Install [Helm](https://helm.sh/docs/intro/install/), [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/) and [Lima](https://github.com/lima-vm/lima#installation) according to the documentatioon. 
+
+* Use Lima to create a K3s cluster. 
+
+    ```plain
+    limactl start template://k3s
+    ```
+
+    When prompted choose "Proceed with current configuration"
+
+* Export the Kubeconfig file as shown in the Lima output
+
+    **Note**:  don't copy and paste from below, copy and paste from your terminal window
+
+    For example:
+    ```plain
+    export KUBECONFIG="/Users/falco.user/.lima/k3s/copied-from-guest/kubeconfig.yaml"
+    ```
+
+* Check to see that the Kubernetes cluster is up and runnnig
+
+    ```plain
+    kubectl get nodes
+    ```
+
+### Create a K3 cluster on Windows or Linux with Virtual box
+
+**Note**: while you can install K3s natively on Linux, the instructions below have been tested and offer a higher likliehood of success. 
+
+* Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads) and [Vagrant](https://developer.hashicorp.com/vagrant/downloads)
+
+* Issue the following commands from the command line to create an Ubuntu 20.04 virtual machine
+
+    ```plain
+        vagrant init bento/ubuntu-20.04
+        vagrant up
+    ```
+
+* Log into the newly launched virtual machine  (the default password is *vagrant*)
+
+    ```plain
+        ssh -p 2222 vagrant@127.0.0.1
+    ```
+
+* Install [Helm](https://helm.sh/docs/intro/install/), [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/), and [K3s](https://docs.k3s.io/quick-start per the instructions for Ubuntu 20.04
+
+    **Note**: be sure you're installing inside your SSH session, not on your local machine. Also installing K3s last makes the next step easier. 
+
+* Export the Kubeconfig file as shown in the K3s output
+
+    **Note**:  don't copy and paste from below, copy and paste from your terminal window
+
+    For example:
+
+    ```plain
+    export KUBECONFIG="/Users/falco.user/.lima/k3s/copied-from-guest/kubeconfig.yaml"
+    ```
+
+* Check to see that the Kubernetes cluster is up and runnnig
+
+    ```plain
+    kubectl get nodes
+    ```
+
+>Note: For Mac users do the remaining steps from the terminal on your local machine. For Windows and Linux users, do these steps in the terminal session for your Virtualbox VM. 
+
+## Install Falco using Helm
+Helm is probably the simplest way to deploy Falco and Falcosidekick. In this section you'll add the Falco Helm chart repository. Next you'll deploy Falco and Falcosidekick with the eBPF probe. Becuse we want to send output to a Slack channel you will provide the Slack webhook information. Additionally, we supply a custom parameter (*user*) that you will customize to your username. 
```

Comment:
```suggestion
Helm is the simplest way to deploy Falco and Falcosidekick. In this section, you'll add the Falco Helm chart repository. Next, you'll deploy Falco and Falcosidekick with the eBPF probe. Because we want to send output to a Slack channel, you will provide the Slack webhook information. Additionally, we supply a custom parameter (*user*) that you will customize to your username. 
```